faq.md

```diff
@@ -0,0 +1,46 @@
+### What is the Jetpack Start API? ###
```

Comment:
A couple of other notes. My markdown linter is complaining about:

- The file doesn't start with an `h1` header
  - After fixing this, the next thing would be to convert `###` to `##` so that the linter doesn't complain about incrementing headers by more than 1 at a time
- There are no blank lines below each header